test/integration/fn_minikube_config.go

```diff
@@ -0,0 +1,47 @@
+// +build integration
+
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"testing"
+)
+
+func testMinikubeConfig(t *testing.T) {
```

Comment:
better to called it testConfig, it is implied we are testing minikube.
we try to never ever Capitalize minikube :)

Reply:
same as file name f_config is good

Reply:
you mentioned `ReadConfig` could use a better name because it is not a profile test, that's why I named this `testMinikubeConfig`, was actually planning to change `ReadConfig` to `ReadMinikubeConfig`.